src/options/containers/App.js

```diff
@@ -0,0 +1,56 @@
+import React, { useEffect, useState } from "react";
+import App from "../components/App";
+import { promisifiedChromeStorageSyncGet } from "../../shared/chrome";
+
+const defaultOptions = {
+  json: {
+    config: "",
+    enable: false
+  },
```

Comment:
If we remove this from `defaultOptions`, we can store the JSON directly in `defaultOptions.options` and use the same value's truthiness to check its existence.

Reply:
Not sure I follow - mind explaining what you mean?

Reply:
As a follow up, I don't think we want to store this config alongside the other Prettier options, because we pass those directly to Prettier when we format code.